src/scripts/install-setup.sh

```diff
@@ -2,23 +2,47 @@ if [[ $EUID == 0 ]]; then export SUDO=""; else # Check if we are root
   export SUDO="sudo";
 fi
 
+SetupPython() {
+    # setups pyenv on a debian based system
+    # these are the system level deps for pyevn install, these are offically recommneded for install by AWS
+    $SUDO apt-get -qq update > /dev/null
+    $SUDO apt-get -qq -y install build-essential zlib1g-dev libssl-dev libncurses-dev libffi-dev libsqlite3-dev libreadline-dev libbz2-dev
```

Comment:
I think this should move back to the install EBCli function just before we run the actual install at L44